### PR TITLE
Update First TC, AWS CD, NCC, ICS conferences

### PR DESCRIPTION
**Pull request by roommen**
Multiple conferences were included in the PR and all details mentioned in the files :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,16 @@ To add a conference to the list, please [contribute](#contributing)!
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [Great International Developer Summit](https://developersummit.com/india/) | 20-24 Apr | IISc, Bengaluru | Polyglot conference and expo series | No |
+| [Namma Cloud Conference](https://konfhub.com/ncc/index.html) | 06 Mar | Radisson Blu, Bangalore | The only event that will help you to get a run-down on everything cloud native | No |
```
Comment (nikhita) [inline README.md]: This conference doesn't have a Code of Conduct (CoC)

File: README.md [hunk 2]
```diff
@@ -9,12 +9,16 @@ To add a conference to the list, please [contribute](#contributing)!
...
 | [GraphQL Asia](https://www.graphql.asia/) | 20-22 Feb | Bengaluru | Largest gathering of the GraphQL community in Asia | [Yes](https://hasurahq.typeform.com/to/ukj62Q) |
 | [Kubernetes Forum Delhi](https://events19.linuxfoundation.org/events/kubernetes-forum-delhi-2019/) | 20-21 Feb | Delhi | Kubernetes Forums connect international and local experts in global cities with adopters, developers, and practitioners of Kubernetes. | [Yes](https://events.linuxfoundation.org/kubernetes-forum-delhi/attend/diversity-scholarships/) |
 | [Kubernetes Forum Bengaluru](https://events19.linuxfoundation.org/events/kubernetes-forum-bengaluru-2019/) | 17-18 Feb | Bengaluru | Kubernetes Forums connect international and local experts in global cities with adopters, developers, and practitioners of Kubernetes. | [Yes](https://events.linuxfoundation.org/kubernetes-forum-bengaluru/attend/diversity-scholarships/) |
+| [India Cloud Summit](https://www.indiacloudsummit.com/) | 14 Feb | Radisson Blu, Bangalore | Multi-cloud one stop to explore new ideas & learn from industry experts | No |
```
Comment (nikhita) [inline README.md]: This conference doesn't seem to accept CFPs and doesn't have a CoC

File: README.md [hunk 3]
```diff
@@ -9,12 +9,16 @@ To add a conference to the list, please [contribute](#contributing)!
...
+| [AWS Community Day](http://awscommunitydaypune.tech/) | 07-08 Feb | Hyatt Regency, Pune | Flagship AWS community cloud conference organized by AWSUGPUN | No |
```
Comment (nikhita) [inline README.md]: This conf doesn't have a CoC

File: README.md [hunk 4]
```diff
@@ -9,12 +9,16 @@ To add a conference to the list, please [contribute](#contributing)!
...
 | [Distributed Systems Conf](https://dsconf.in/) | 01 Feb | Hotel Novotel, Viman Nagar, Pune | India’s foremost conference on Distributed Systems | No |
 | [Machine Learning Developers Summit](https://www.mlds.analyticsindiasummit.com/) | 30-31 Jan | Indian School of Business, Hyderabad | India’s leading Machine Learning innovators and practitioners to share their ideas and experience about machine learning tools, advanced development & developer products | No |
 | [Machine Learning Developers Summit](https://www.mlds.analyticsindiasummit.com/) | 22-23 Jan | NIMHANS Convention Center, Bengaluru | India’s leading Machine Learning innovators and practitioners to share their ideas and experience about machine learning tools, advanced development & developer products | No |
+| [FIRST Technical Colloquium](https://www.first.org/events/colloquia/bangalore2020/) | 21-23 Jan | Cisco Campus, Bengaluru | Information and product security conference | No |
```
Comment (nikhita) [inline README.md]: This conf doesn't have a CoC

**Comment by roommen** [comment]
oh ok, thanks Nikhita for letting me know about the CoC aspect. I guess, will close this PR

**Pull request closed without merging**